Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+            <figcaption>
+                Mac Touch Bar showing editing buttons
+            </figcaption>
+        </figure>
+        <figure>
+            <img src="images/ios-editing-buttons.png" width="300" alt="iOS selection menu showing editing buttons">
+            <figcaption>
+                iOS selection menu showing editing buttons
+            </figcaption>
+        </figure>
+        <p>
+            To address the issue, a new contenteditable-disabled attribute can be applied as shown in the example below.
+        </p>
+        <pre class="example">
+            &lt;div contenteditable="true" contenteditable-disabled="
+                formatbackcolor
+                formatdent
+                formatfont
+                formatjustify
+                formatscript
+                formatstrikethrough
+                formatunderline
+                inserthorizontalrule">
+                This is a contenteditable region of a document that supports:
+                &lt;ul>
+                    &lt;li>bold text&lt;/li>
+                    &lt;li>italic text&lt;/li>
+                    &lt;li>links&lt;/li>
+                    &lt;li>lists&lt;/li>
+                    &lt;li>and possibly other formatting outside the scope of this specification.&lt;/li>
+                &lt;/ul>
+                
+                This contenteditable region explicitly does not support:
+                &lt;ul>
+                    &lt;li>background colors&lt;/li>
+                    &lt;li>indentation&lt;/li>
+                    &lt;li>font name, size or color customization&lt;/li>
+                    &lt;li>justification&lt;/li>
+                    &lt;li>superscript or subscript text&lt;/li>
+                    &lt;li>strikethrough&lt;/li>
+                    &lt;li>underlines&lt;/li>
+                    &lt;li>horizontal rules&lt;/li>
+                &lt;/ul>
+            &lt;/div>
+        </pre>
+    </section>
+    <section id="contenteditable-disabled">
+        <h2>
+            <code>contenteditable-disabled</code> DOM Content Attribute.
+        </h2>
+        <p>
+            The <dfn><code>contenteditable-disabled</code></dfn> attribute indicates which user agent-supplied editing UI SHOULD be disabled.  If specified, the attribute must have a value that is an <a href=https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#unordered-set-of-unique-space-separated-tokens>unordered
+            set of unique space-separated tokens</a> [[HTML]] which
+            are <a href=https://infra.spec.whatwg.org/#ascii-case-insensitive>ASCII case-insensitive</a> [[INFRA]].
+        </p>
+        <p>
+            The allowed set of tokens are listed below.  Each token matches the name of an <a href=https://www.w3.org/TR/input-events-1/#events-inputevents>Input Type</a> [[input-events-1]], or is a shortcut that is equivalent to specifying a set tokens.
```

Comment:
Where possible, please alway avoid `<a href>` - worst case, use `data-cite=` instead. 
```suggestion
            The allowed set of tokens are listed below.  Each token matches the name of an <a data-cite="input-events-1#events-inputevents">Input Type</a>, or is a shortcut that is equivalent to specifying a set tokens.
```